src/middleware/compress/index.ts

```diff
@@ -0,0 +1,24 @@
+import * as WebStreams from 'node:stream/web'
```

Comment:
@tani 

Hey, we must not use **node** module. Only use web standard API available on Cloudflare and Deno.

Reply:
Noted. Cheers.